src/platforms/dotnet/guides/xamarin/common/troubleshooting.mdx

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This file isn't displaying in preview. Revise it to `common/troubleshooting/index.mdx`.

Reply:
Moving to `common/troubleshooting/index.mdx` didn't help.
But leaving the doc at the original place and adding troubleshooting file to .NET fixed it, but then, the .NET doc ended up with a blank troubleshooting page while the Xamarin one got proper troubleshooting 😅.
So I believe it'll be fine to leave all the Xamarin troubleshooting info into its own index file.

Reply:
Something is unfortunately awry here. The troubleshooting information is now displaying on the Getting Started page, which is not the way our content works. The information architecture is designed to get users up and running (the primary index page for the platform), with sections then available for configuration, usage, and other key elements (such as source maps for JS). 

We can't put the troubleshooting information on the primary index page of the platform; it should be its own page. I'm happy to help you if you get stuck enabling this.

Reply:
I accept some help @PeloWriter 😄
Long short story, I was trying to follow the layout chosen for Unity, https://docs.sentry.io/platforms/unity/troubleshooting/,
 but Unity is a `platform` and not a `guide` so things got tricky for me.